index.js

```diff
@@ -208,6 +435,17 @@ module.exports = (opts = {}) => {
         rule.raws.semicolon = true
         if (rule.nodes.length === 0) rule.remove()
       }
+    },
+
+    RootExit(root) {
+      if (root[hasRootRule]) {
+        root.walk(node => {
```

Comment:
[`walkAtRules()`](https://postcss.org/api/#root-walkatrules) will be readable and could be even a little faster